griptape/artifacts/base_artifact.py

```diff
@@ -50,5 +49,8 @@ def __bool__(self) -> bool:
     def __len__(self) -> int:
         return len(self.value)
 
+    def to_bytes(self) -> bytes:
+        return self.to_text().encode(encoding=self.encoding, errors=self.encoding_error_handler)
```

Comment:
This is basically for code reuse within the most common "Text-like Artifacts"? 

How would/when would I set encoding_error_handler? Have you seen the need arise?

Reply:
> This is basically for code reuse within the most common "Text-like Artifacts"?

Most immediately, it's needed to keep this behavior [here](https://github.com/griptape-ai/griptape/blob/35e2461cfd468bb54d5c9d793b29cf3d9bd90d95/griptape/mixins/artifact_file_output_mixin.py#L45).

> How would/when would I set encoding_error_handler? Have you seen the need arise?

It's definitely an edge case field, you can see an example [here](https://stackoverflow.com/questions/3224268/python-unicode-encode-error). I have not run into a use myself, but it was there before so I kept it.